### ISSUE TITLE
User-added pools?

### ISSUE THREAD

**Issue by bigreddmachine** [micro-machine/monero-monitor#6]
Re: #3 by @kenshi84 

> Perhaps a better way could be to let users add/delete pool addresses as they wish.

Since it's not immediately obvious how to get the right setting for a pool's API, I'm not sure that user's adding pools is a great option. How could we make it straightforward for users?

**Comment by kenshi84** [comment]
Yes, that's a tricky issue. And I myself didn't know how to access APIs of xmrpool.eu and supportxmr.com. I managed to add them to the list by simply copying the settings of other pools (i.e. the port 8117), so maybe it's still reasonable to let the user add pools at their own will by experimenting the right setting while referring to the existing predefined settings. If unsure, users could just contact the pool maintainers.

I'd personally appreciate such a functionality, at least :)
